config.hcl

```diff
@@ -0,0 +1,43 @@
+consul {
```

Comment:
I dont think we need this file anymore sense we wont be building locally

Reply:
yeah we could remove it. Hat that the config is only found in K8s stuff tough. But guess that the way it be. I am indifferent.